mycroft/client/wifisetup/main.py

```diff
@@ -135,38 +208,145 @@ def save(self):
 
 
 class WiFi:
-    NAME = "WiFiClient"
-
     def __init__(self):
         self.iface = pyw.winterfaces()[0]
         self.ap = AccessPoint(self.iface)
         self.server = None
         self.client = WebsocketClient()
         self.enclosure = EnclosureAPI(self.client)
-        self.config = ConfigurationManager.get().get(self.NAME)
         self.init_events()
-        self.first_setup()
+        self.threadConMon = None
+        self.threadConMon_stop = threading.Event()
 
     def init_events(self):
         self.client.on('mycroft.wifi.start', self.start)
         self.client.on('mycroft.wifi.stop', self.stop)
         self.client.on('mycroft.wifi.scan', self.scan)
         self.client.on('mycroft.wifi.connect', self.connect)
 
-    def first_setup(self):
-        if str2bool(self.config.get('setup')):
-            self.start()
-
     def start(self, event=None):
+        # Fire up the MYCROFT access point for the user to connect to
+        # with a phone or computer.
         LOG.info("Starting access point...")
-        self.client.emit(Message("speak", metadata={
-            'utterance': "Initializing wireless setup mode."}))
+
+        # zap any existing leases file because:
+        # a) we don't care about previous leases
+        # b) we will monitor it for connections
+        # os.remove('/var/lib/misc/dnsmasq.leases')
+
+        # Fire up our access point
         self.ap.up()
         if not self.server:
+            LOG.info("Creating web server...")
```

Comment:
Those were needed to debug a situation where the self.server.start() was not firing.  Given how rarely this process will be invoked by a user, having extra logs to debug situations where it isn't working is worth it.  This doesn't run constantly and the logs aren't going to overflow from a couple extra messages here.